media-querys.css

```diff
@@ -0,0 +1,193 @@
+/* Big screens */
+@media (min-width: 768px) {
+  header.nav {
+    margin-left: 24px;
+  }
+
```

Comment:
Kindly ensure that you keep to the margin between the social icons in the footer.

Your design:
![image](https://user-images.githubusercontent.com/20554583/229232401-bf53063d-69d5-4cb9-876c-4729cd555b15.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/229232455-a8beecfa-cb3e-4d98-b41b-168ab43d755f.png)

Reply:
It's Done